### DISCUSSION TITLE
Flash Floppy does not recognize Flash drive after a few insertions

### DISCUSSION THREAD

**Discussion by DarrellH89** [keirf/flashfloppy#900] [Q&A]
I'm using Flash Floppy, ver 3.42, on two Heathkit H-89's. I'm developing code on one system and testing on the other. The test files are copied to an image on the flash drive and moved to the other system. After a few swaps, Flash Floppy doesn't register that a USB drive is inserted. I need to cycle the system power to access the USB drive. Is there way to cycle the Flash Floppy without shutting the system down?

**Comment by keirf** [comment]
Never heard of this failure before. Is it true of both Goteks? And the failure mode is that the FlashFloppy banner remains (do these Goteks have OLED display)?

There should be no need for a reset option, so there isn't one in the main menu.

**Reply by DarrellH89** [reply]
Yes, it happens on both. The FlashFloppy banner is displayed as if no USB drive was present. They are running with an OLED using "display-type = oled-128x64-rotate". The only solution I have is to cycle the system power.

**Reply by keirf** [reply]
Can you make it happen simply by removing and inserting the usb drive a few times in quick succession from power on? When this problem occurs can you enter the main menu (by pressing the rotary encoder)? If you reinsert the usb drive when this problem occurs does the Gotek remain stuck? Have you tried another usb drive? And what is the model number of your Goteks as printed at rear of their PCB (alternatively, what does MCU Info report in main menu)?

**Reply by DarrellH89** [reply]
Now that you asked for details the issue fails to appear. I swapped the Flash drive at least 20 times with no failure. I'll go back to working on the code and provide the details you requested if I get a repeat. Before I posted it happened on both systems at least three times. At least once, I pressed the rotary encoder with no change in the display. Thanks for your reply

**Reply by keirf** [reply]
Thanks. A complete hang is definitely weird of course. I wonder if it is stick or voltage related.
